metrics/interface/src/main/assembly/resources/conf/iotdb-metric.yml

```diff
@@ -40,8 +43,6 @@ pushPeriodInSecond: 5
 #                                                      #
 # if the reporter is prometheus,                       #
 # then the following must be set.                      #
-# if you use dropwizard, all below                     #
-# should be the config of pushgateway                  #
 #                                                      #
 ########################################################
 prometheusReporterConfig:
```

Comment:
Since we removed the push_gateway, the prometheusExporterUrl becomes useless.
Then all the three lines could be replaced by a simple config:  `prometheusExporterPort: 9091`.

Reply:
yeap, that's right and I will fix it

Reply:
The `pushPeriodInSecond` in line 40 also becomes useless

Reply:
Oh, `pushPeriodInSecond` is used in IoTDB Reporter

Reply:
This has been finished.